tools/test

```diff
@@ -0,0 +1,5 @@
+#!/bin/sh
```

Comment:
I was planning to add something like `sigh webtest` to run all the `*-webtest.js` tests, but I suppose this is OK for now.

Reply:
I'd support reworking this so that our README just directs folks to run `./tools/sigh install` and `./tools/sigh test` and it does all of the things for everything, but I'm not sure whether that's directionally a goal for sigh, so in meantime I figure at least we can live with this and dream...